calculator.c

```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
```

Comment:
Are you able to run your code? Your code had a lot of syntax errors. You should come see me or Ms. Hsu.